### ISSUE TITLE
execFile's callback is never called

### ISSUE THREAD

**Issue by strager** [oven-sh/bun#5441]
### What version of Bun is running?

1.0.1+31aec4ebe325982fc0ef27498984b0ad9969162b

### What platform is your computer?

Linux 5.19.0-46-generic x86_64 x86_64

### What steps can reproduce the bug?

```javascript
import child_process from "node:child_process";
console.log('starting');
child_process.execFile('true', (err, stdout, stderr) => {
  console.log('done');
});
```

### What is the expected behavior?

```shell
$ node exec.mjs    
starting
done
```

### What do you see instead?

```shell
$ bun run exec.mjs    
starting
```

**Comment by Hanaasagi** [comment]
I think #5481 has fixed this issue. `execFile` uses the `close` event to execute the `callback`.

https://github.com/oven-sh/bun/blob/28f345346650dc94330b4a2cdcd1297df2916a39/src/js/node/child_process.js#L445-L446

**Comment by Hanaasagi** [comment]
Fixed in 1.0.3, please upgrade. Thankes.
